packages/p2p/src/index.js

```diff
@@ -1 +1,2 @@
 export { default as P2P } from './components/app.jsx';
+export { default } from './components/app.jsx';
```

Comment:
Agree with @shayan-deriv here. Why not change it in `packages/cashier/src/constants/routes-config.ts` to named import?

Reply:
Yes @nijil-deriv we can pick from both of the solution, there's no right or wrong here, we can all agree with one approach and go with it.

Personally, I think if we are importing this main `P2P` component and there's nothing else exported from `p2p/src/index.js` we should go with default export, it looks good to me.

but anyway let's see if all agrees with either named or default export I can change it accordingly

cc: @ali-hosseini-deriv @shayan-deriv @niloofar-deriv @farrah-deriv